[2305]: ath79: Revert "ath79: calibrate dlink dir-825 b1 with nvmem"

Lucky1openwrt opened the pull request:
DIR-852-B1 starts with random LAN & WAN MAC addresses
reverting this commit makes it the same as it's working twin
ar7161_trendnet_tew-673gru.dts

snapshot MAC addresses work this is only to fix V23.05

This reverts commit https://github.com/openwrt/openwrt/commit/de3d60b982c7b8e1821b90b312db57287e74ede3.

rmilecki reviewed (changes requested):
I'd really expect and explanation why do we need to revert it.

Also if it's basically a revert, then maybe you could use `git revert`?

Lucky1openwrt commented:
well as the wan and lan mac addresses are just random at the moment & have been at lest since 25.03.0
it would be nice not having to manually set them each time I erase settings
I don't know how at the moment to revert I will have to look it up 11:43PM atm so tomorrow

Lucky1openwrt committed:
[2305]: ath79: Revert "ath79: calibrate dlink dir-825 b1 with nvmem"

DIR-852-B1 starts with random LAN & WAN MAC addresses
reverting this commit makes it the same as it's working twin
ar7161_trendnet_tew-673gru.dts

snapshot MAC addresses work this is only to fix V23.05

This reverts commit de3d60b982c7b8e1821b90b312db57287e74ede3.

Signed-off-by: Alan Luck <luckyhome2008@gmail.com>

Lucky1openwrt commented:
@rmilecki 
I think this is what you ment

rmilecki commented:
I was mostly wondering why does it happen.

Did you try cherry picking commit daaa0c1b2504 ("`ath79: replace "mac-address-ascii" with "mac-base"`") instead of reverting this change?

Lucky1openwrt commented:
that commit in not part of V23.05
the main snapshots MAC addresses works so nothing to change there 
* the wan port didn't operate atm but  it's MAC address was correct 

the addresses are ok on both wifi bands and so most of it works
just for this version seems easer to revet as I don't know why it's not
but it's twin with less leds is still working so lets just make it the same

rmilecki commented:
> that commit in not part of V23.05

I assume you mean commit daaa0c1b2504 ("`ath79: replace "mac-address-ascii" with "mac-base"`"). I know it's not part of `openwrt-23.05` (it wasn't backported yet).
There is some chance it fixes your problem. So I asked you to cherry pick it (`git cherry-pick -x daaa0c1b2504`) instead of reverting another commit and see if that helps with reported problem.

I wasn't talking about doing any `master` / `main` changes.

Lucky1openwrt commented:
I did manually edit the .dst with a few options & it just made it worse "lost the wifi MAC's well "
but I can specificity test that if you would like

rmilecki commented:
Obsoleted by commit 9276bb1cf590 ("`ath79: replace "mac-address-ascii" with "mac-base"`").

Thank you for testing!

Lucky1openwrt commented:
thanks :)
